ucsb-cs156-f24/proj-happycows-f24-12
# Fixed Incorrect Capitalization

## Overview
In this PR, I fixed the capitalization of "day" on the main page to be "Day" and also the word "cows" on the buttons "Buy Cows" and "Sell Cows".

## Screenshots
<img width="1128" alt="image" src="https://github.com/user-attachments/assets/7049fc94-95bd-4662-b594-f2ec57ded3ed">

## Validation
1. Download this branch.
2. Run the project.
3. Check main page for changes

## Tests

## Linked Issues
Closes #59
